src/Gui/ViewProviderLink.cpp

```diff
@@ -1801,6 +1801,10 @@ void ViewProviderLink::updateData(const App::Property *prop) {
     return inherited::updateData(prop);
 }
 
+static inline bool canScale(const Base::Vector3d &v) {
+    return fabs(v.x)>1e-7 && fabs(v.y)>1e-7 && fabs(v.z)>1e-7;
```

Comment:
Do we have a constant we can use here, rather than the hardcoded 1e-7?

Reply:
Precision::Confusion() ?